docs/platform_operators/rotating-secrets-and-certs.md

```diff
@@ -2,7 +2,54 @@
 Most secrets in cf-for-k8s can be rotated by simply changing the values in your `cf-values.yml` file and running a standard deploy using ytt and kapp. The rotation is complete when the kapp deploy succeeds.
 
 ## Exceptions
-As of September 2020, it is currently not possible to rotate `app_registry` credentials. Future work on this will be tracked by https://www.pivotaltracker.com/story/show/173090115
+
+The following fields currently cannot be rotated:
+
+* `blobstore.secret_access_key`
+* `capi.database.encryption_key`
+* `capi.database.password`
+* `cf_admin_password`
```

Comment:
this one can be rotated, right?

Reply:
Do we want to update the `## Exceptions` section in general to indicate the context in which these rotations fail? It seems like a subset of the rotations are failing for uptimer validations but actually work manually?